example/simple-server/README.md

```diff
@@ -0,0 +1,120 @@
+# Overview
+This example is a simple server showing how to create server side code in an OpenZiti Network. A server is spun up to 
+serve plain HTTP requests and a Zitified server is spun up to serve zitified requests. This example attaches one server 
+to the underlay (HTTP server) and a second server on the overlay (Ziti Service).
+
+This example demonstrates:
+* Binding a service and listening for HTTP connections
+* Accessing the service via a tunneler
+  * [Ziti Desktop Edge for Windows](https://github.com/openziti/desktop-edge-win/releases)
```

Comment:
this could just be one link i think? https://openziti.github.io/ziti/clients/which-client.html